src/config/preferences.h

```diff
@@ -42,7 +42,7 @@
 #include <glib.h>
 
 #define PREFS_MIN_LOG_SIZE 64
-#define PREFS_MAX_LOG_SIZE 1048580
+#define PREFS_MAX_LOG_SIZE 10485760
```

Comment:
```suggestion
#define PREFS_MAX_LOG_SIZE (10 * 1024 * 1024)
```

but that's personal preference, you can also keep it as is.